index.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+<link rel="stylesheet" href="https://cdn.linearicons.com/free/1.0.0/icon-font.min.css">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<title>Login</title>
+
+<!--CSS-->
+<link rel="stylesheet" href="css/main.css">
+<link href="https://fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i%7CPlayfair+Display:400,400i,700,700i,900,900i" rel="stylesheet">
+<link rel="stylesheet" href="https://cdn.linearicons.com/free/1.0.0/icon-font.min.css">
+<link rel="stylesheet" href="assets/pe-icon-7-stroke/css/pe-icon-7-stroke.css">
+
+</head>
+
+<body>
+
+	<div class="loginCon"> 
+		<div class="row">
+			<!--LeftSide-->
+			<div class="left-log column-large-6 column-medium-5 show-for-large align-center cover-content">
+                <div class="bgimg"></div>
+			</div>
+ 
+            <!--RightSide-->
+            <div class="right-log column-large-6 column-medium-5 column-small-8 align-center cover-content">
+                <div class="whiteLogo show-for-small" alt="white">
+                <img src="images/LogoWhite.svg" alt="logowhite">
+            </div>
+                <div class="vertical-logo align-center show-for-medium">
+                     <img src="images/logo.svg" alt="logo">
+                </div>
+            <div class="loginForm">
+            <form name="login-form" id="login-form" class="contact-form js-animation" method="pos"> 
```

Comment:
Even tho this is not here, and the form will not be functional, there is no such thing as method="pos", it should be **post**.